constructiveness_toxicity_crowdsource/common/data_input_output_functions.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import sys
+import glob
+import ntpath
+
+def get_full_annotation_csv(data_path, batch):
+    '''
+    :data_path: (str) the path of the directory containing annotated data CSVs
+    :batch: (int) annotation batch number
+    :return: (str) the path containing the full annotation CSV
+    '''
+    
+    input_csv_files = [f for f in glob.glob(data_path + 'batch' + str(batch) + '_f*.csv') 
+                                               if ntpath.basename(f).count('_') == 1]
+    if len(input_csv_files) > 1:
+        print('Please review your input dir. There seem to be more than one input files. ')
```

Comment:
I'm confused by this check. Could you clarify in the function comment if this function expects just one input csv or many? Also it's better to raise an exception than to print a line here. If we fail, we want to fail loudly.